### DISCUSSION TITLE
Adopt a project

### DISCUSSION THREAD

**Discussion by JarbasAl** [OpenVoiceOS/ovos-buildroot#169] [Open Discussion]
Some chats with community members gave me the idea to put up a list of tasks, "here is a cool idea we do not have time to do ourselves"

Please provide any feedback and suggest more things, think of this as a decentralized TODO list anyone can help with

# Maintainers wanted

projects we are no longer maintaining due to lack of time / expertise (not obsolete)

- https://github.com/OpenVoiceOS/community-docs - community maintained documentation
- https://github.com/OpenVoiceOS/Open-Linguistika - QML translation tool, can be expanded for skill resource files
- https://github.com/OpenVoiceOS/ovos-qubes - hardened ovos setup for Qubes  (I'm no longer using Qubes)
- [x] https://github.com/OpenVoiceOS/ovos-PHAL-plugin-homeassistant adopted by @mikejgray

# Documentation Missing

- help new users, send a PR to https://github.com/OpenVoiceOS/community-docs
- add a README.md. eg https://github.com/OpenVoiceOS/ovos-vad-plugin-silero/issues/2

# Port a skill

old skills needing to be ported to latest best practices

note: any skills from JarbasSkills that are updated will be moved to OpenVoiceOS org

- [x] https://github.com/OpenVoiceOS/ovos-skill-easter-eggs adopted by @mikejgray
- [x] https://github.com/OpenVoiceOS/skill-wallpapers  <- integrate with [wallpaper api](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager),  adopted by @OpenVoiceOS
- https://github.com/JarbasSkills/skill-wikihow
- https://github.com/JarbasSkills/skill-email-commands
- https://github.com/JarbasSkills/skill-parrot
- https://github.com/JarbasSkills/skill-laugh
- https://github.com/JarbasSkills/skill-xkcd <- integrate with [wallpaper api](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager)
- https://github.com/JarbasSkills/skill-voip
- https://github.com/JarbasSkills/skill-moon-game
- https://github.com/JarbasSkills/skill-apod<- integrate with [wallpaper api](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager)
- https://github.com/JarbasSkills/skill-countries
- https://github.com/JarbasSkills/skill-natgeo-pod<- integrate with [wallpaper api](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager)
- https://github.com/JarbasSkills/skill-blue-marble<- integrate with [wallpaper api](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager)
- https://github.com/JarbasSkills/skill-iss-location<- integrate with [wallpaper api](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager)
- https://github.com/JarbasSkills/skill-helioviewer<- integrate with [wallpaper api](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager)
- https://github.com/JarbasSkills/skill-pickup-lines
- https://github.com/JarbasSkills/skill-quote-of-day
- https://github.com/JarbasSkills/skill-algorithmic-art <- integrate with [wallpaper api](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager)
- https://github.com/JarbasSkills/skill-confucius-quotes
- https://github.com/JarbasSkills/skill-konami-code
- https://github.com/JarbasSkills/skill-good-manners
- [x] https://github.com/OpenVoiceOS/skill-ovos-dictation <- adopted by @OpenVoiceOS
- https://github.com/JarbasSkills/cryptocoin_skill
- https://github.com/JarbasSkills/skill-enclosure-ctrl <- mk1 phal plugin companion skill
- https://github.com/domcross/severe-weather-information-skill
- https://github.com/gras64/learning-skill
- https://github.com/TREE-Ind/Wiktionary-Skill
- https://github.com/retrodaredevil/pokemon-skill
- https://github.com/luke5sky/discontinued-remember-skill
- https://github.com/austin-carnahan/days-in-history-skill
- [x] https://github.com/mikejgray/meal-plan-skill author still active @mikejgray
- https://github.com/blue1stone/bring-shopping-lists-skill
- https://github.com/brezuicabogdan/quiet-hours-skill
- https://github.com/samhithatarra/horoscope-skill
- https://github.com/TheLastProject/travel-time-skill
- https://github.com/mofadashi/mycroft-skill-allergy
- https://github.com/traverseda/mycroft-skill-unitconversion

# create a plugin

- https://github.com/orgs/OpenVoiceOS/discussions/159 - STT Request - Azure 
- https://github.com/orgs/OpenVoiceOS/discussions/153 - STT Request - KaldiGstreamerServer 
- https://github.com/orgs/OpenVoiceOS/discussions/148 - STT Request - deepspeech server 
- https://github.com/orgs/OpenVoiceOS/discussions/147 - STT + TTS Request - Yandex 
- https://github.com/orgs/OpenVoiceOS/discussions/146 - STT Request - Google Cloud 
- https://github.com/orgs/OpenVoiceOS/discussions/157 - WIFI request: qr code 
- https://github.com/orgs/OpenVoiceOS/discussions/150 - TTS Request - Mozilla TTS Server 
- https://github.com/orgs/OpenVoiceOS/discussions/151 - STT+TTS Request: elhuyar (basque) 
- https://github.com/orgs/OpenVoiceOS/discussions/143 - TTS Request - Dilmanc TTS (Azerbaijan) 
- https://github.com/orgs/OpenVoiceOS/discussions/145 - STT+TTS Request: VK Cloud (russian)

**Comment by ShuffleBit** [comment]
> old skills needing to be ported to latest best practices

Newb question: where are the latest best practices documented? Is that updated regularly?

**Reply by builderjer** [reply]
Docs are updated when someone can get to it. Bet way to get an idea is to look at one of the skills that are already updated.

**Reply by ShuffleBit** [reply]
Thank you. I kind of assumed that based on my own love of documenting things.

**Comment by mikejgray** [comment]
I'll go ahead and bring this demo skill up to date for OVOS: https://github.com/mikejgray/meal-plan-skill

**Reply by mikejgray** [reply]
https://github.com/mikejgray/skill-meal-plan/pull/1

**Reply by mikejgray** [reply]
https://github.com/mikejgray/skill-meal-plan v1.0.0 is now available for UAT @JarbasAl 
https://pypi.org/project/meal-plan-skill/1.0.0/
